### PR TITLE
Adding curl commands for each route in routes.rb

### PR DESCRIPTION
**Pull request by nw0428**
LGTU?

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -13,27 +13,59 @@
...
+
+  # Example URL: localhost:3000/example3?number=100&number2=11
+  get 'but-why', to: "example#proof_you_can_name_this_whatever_123_abc"
+  # curl localhost:3000/but-why
+
 
   # More advanced ruby examples
   get 'rock_paper_scissors', controller: 'games', action: 'rock_paper_scissors'
-  get 'state_search', controller: 'search', action: 'state_filter'
+  # curl localhost:3000/rock_paper_scissors?choice=rock
```
Comment (toppsdown) [inline config/routes.rb]: oh haha, I only meant for the HTTP Verbs section above.  I don't think we need curls for all of them

**Comment by toppsdown** [comment]
Made a comment, but I don't think we need curl for every single example, just the HTTP Verb ones.  Otherwise 👍

File: config/routes.rb [hunk 2]
```diff
@@ -13,27 +13,59 @@
...
+  # curl - X PUT localhost:3000/example1
+
   post 'example1', to: "example#example1"
+  # curl - X POST localhost:3000/example1
+
   delete 'example1', to: "example#example1"
+  # curl - X DELETE localhost:3000/example1
+
+  # These examples use different HTTP verbs to route to different methods
+  put 'example', to "example#example_put"
```
Comment (chdastolfo) [inline config/routes.rb]: Syntax (to:) here and in the example below.

**Comment by toppsdown** [comment]
I made some changes, gonna merge

**Pull request merged**